src/site/content/en/blog/sharedarraybuffer-updates-january-2021/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Firefox and Chrome now support SharedArrayBuffer (again) for cross-origin isolated pages
+subhead: >
+  The API for sharing memory between the main thread and worker threads is available (again)
+  on the desktop and Android versions of Firefox and Chrome. SharedArrayBuffer is only available for
+  websites that have implemented COOP and COEP.
+description: >
+  The API for sharing memory between the main thread and worker threads is available (again)
+  on the desktop and Android versions of Firefox and Chrome. SharedArrayBuffer is only available for
+  websites that have implemented COOP and COEP.
+authors:
+  - kaycebasques
+date: 2021-01-13
+# updated: 2020-07-22
+hero: hero.jpg
+alt: An illustration of a person browsing a website that has a popup, an iframe, and an image.
+tags:
+  - blog
+  - security
+  - performance
+  - memory
+feedback:
+  - api
+---
+
+Here's some quick browser compatibility updates regarding [`SharedArrayBuffer`][mdn]:
+
+* Chrome recently [announced](TODO) that it's enabling `SharedArrayBuffer` in
+  Android Chrome 88. It will only be enabled for [cross-origin
+  isolated](/coop-coep/) pages.
+* Chrome also announced that, starting in Desktop Chrome 91, it will start
+  restricting `SharedArrayBuffer` to cross-origin isolated pages.
+  Currently it's available for all pages. Sites that can't meet the cross-origin
+  isolation requirements in time can request an origin trial to extend the
+  current behavior until Chrome 93.
```

Comment:
I'd emphasize that this will break existing implementations if they don't opt-in to cross-origin isolation.